setup.cfg

```diff
@@ -38,6 +38,7 @@ install_requires =
 
 [options.extras_require]
 tests =
+    openlineage-airflow>=0.19.2
```

Comment:
This should not be in tests. @phanikumv I think we should have a section for all which should contain open lineage and another section called openlineage like in https://github.com/astronomer/astronomer-providers/blob/main/setup.cfg#L118

Reply:
I'll make the change. I think I copied this from a different link you sent, or maybe just inferred this may work because the tests were breaking without some import.